.release-please-manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "hooks/openfeature-hooks-opentelemetry": "0.1.3",
   "providers/openfeature-provider-flagd": "0.1.5",
-  "providers/openfeature-provider-ofrep": "0.1.0",
```

Comment:
You don't need to bump this manually. It's handled automatically by our release process.